app/views/user/emails/_email.html.slim

```diff
@@ -11,8 +11,11 @@
         = t('.primary')
 
   td
-    - unless email.primary?
+    - if !email.confirmed?
+      = t('.confirmation_sent')
+      = link_to t('.resend_confirmation'), send_confirmation_user_email_path(email), method: :post
```

Comment:
@lowjoel Button does not display very nice here, cause I have text before it. Can we stick to link ? 

![screen shot 2016-02-17 at 6 28 07 pm](https://cloud.githubusercontent.com/assets/4983239/13106634/58808950-d5a4-11e5-9404-cf47309e6df5.png)